src/sentry/receivers/core.py

```diff
@@ -151,14 +156,14 @@ def freeze_option_epoch_for_project(instance, created, app=None, **kwargs):
 )
 
 post_save.connect(
```

Comment:
Note: I had to reorder these, as the epoch needs to be set before the project key is created.